common/src/main/java/com/wynntils/functions/InventoryFunctions.java

```diff
@@ -19,6 +21,162 @@
 import net.minecraft.world.item.ItemStack;
 
 public class InventoryFunctions {
+    public static class AccessoryCurrentDurabilityFunction extends Function<Integer> {
```

Comment:
You only need to have a single getDurability function; we have already helper generic functions for dealing with capped values (to get max and current value).

Reply:
Held item has a function for current, max and CappedValue durability, or are they just outdated now?

Reply:
We should not have individual functions returning the current/max part of capped values -- we should only return the capped value (and the function should not be named "capped"). There are probably a handful of functions that were introduced before we had full support for capped values in functions; ideally these should be removed but maybe we want to keep backwards compatibility.

We should not be adding new such functions.

Or did I misunderstand you?

Reply:
Ah ok, yeah that's what I meant so we're just keeping them so older functions don't break

Reply:
Yep. At some point I'd like to deprecate them and get rid of them. Maybe we need "upfixers" for functions too... or just warn folks that they will disappear in X days time, or whatever.